### ISSUE TITLE
White text on light grey background is very low contrast

### ISSUE THREAD

**Issue by gidsg** [guardian/open-platform-site#11]
![screen shot 2014-07-04 at 11 02 05](https://cloud.githubusercontent.com/assets/1764158/3480535/3eb1e728-0362-11e4-99cb-b89af2db7c1f.png)

As per WCAG/Guardian guidelines, we should be at least 4.5:1 contrast ratio
http://snook.ca/technical/colour_contrast/colour.html

**Comment by mchv** [comment]
I think I have improved it with last changes:
![screen shot 2014-07-08 at 12 11 03](https://cloud.githubusercontent.com/assets/615085/3508820/9af6c11c-0690-11e4-88a1-8452d1ca02ea.png)

**Comment by gidsg** [comment]
fixed
